From 4f8916b90e9dce8d8a820dc4cddce8929bf95664 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:05:08 -0400
Subject: [ethernet] qlcnic: Miscellaneous conversions to ETH_ALEN

Message-id: <1408626422-3941-25-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88566
O-Subject: [PATCH RHEL 7.1 024/138] net:drivers/net/ethernet/qlogic: Miscellaneous conversions to ETH_ALEN
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Joe Perches <joe@perches.com>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id d458cdf712e0c671e8e819abb16ecd6e44f9daec

Convert the memset/memcpy uses of 6 to ETH_ALEN
where appropriate.

Also convert some struct definitions and u8 array
declarations of [6] to ETH_ALEN.

Signed-off-by: Joe Perches <joe@perches.com>
Acked-by: Arend van Spriel <arend@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_hw.c b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_hw.c
index f8adc7b..73e72eb 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_hw.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_hw.c
@@ -445,7 +445,7 @@ int qlcnic_82xx_sre_macaddr_change(struct qlcnic_adapter *adapter, u8 *addr,
 
  mac_req = (struct qlcnic_mac_req *)&req.words[0];
  mac_req->op = op;
- memcpy(mac_req->mac_addr, addr, 6);
+ memcpy(mac_req->mac_addr, addr, ETH_ALEN);
 
  vlan_req = (struct qlcnic_vlan_req *)&req.words[1];
  vlan_req->vlan_id = cpu_to_le16(vlan_id);
-- 
1.7.1